runtime/src/bank/sysvar_cache.rs

```diff
@@ -1,13 +1,23 @@
 use {
-    super::Bank, solana_program_runtime::sysvar_cache::SysvarCache,
-    solana_sdk::account::ReadableAccount,
+    super::Bank,
+    solana_program_runtime::sysvar_cache::SysvarCache,
+    solana_sdk::{account::ReadableAccount, feature_set},
 };
 
 impl Bank {
     pub(crate) fn fill_missing_sysvar_cache_entries(&self) {
         let mut sysvar_cache = self.sysvar_cache.write().unwrap();
         sysvar_cache.fill_missing_entries(|pubkey, callback| {
-            if let Some(account) = self.get_account_with_fixed_root(pubkey) {
+            if solana_sdk::sysvar::last_restart_slot::check_id(pubkey)
```

Comment:
I don't think this feature gate here is necessary.

Reply:
removed the check